Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/install/pre-release.md

> For information on our versioning convention, refer to
> [Components of DVC version](/doc/command-reference/version#components-of-dvc-version).

> To install a development version for contributing to the project, you can refer to the
> [Development environment](/doc/user-guide/contributing/core#development-environment).

Reviewer: ```suggestion
To install a development version for contributing to the project, see the
[Development environment](/doc/user-guide/contributing/core#development-environment)
guide.
```
